test/connectors/UpscanConnectorSpec.scala

```diff
@@ -44,7 +49,7 @@ class UpscanConnectorSpec extends UnitSpec with GuiceOneAppPerSuite with Mockito
             )
         )
 
-        val result = await(connector.getUpscanFormData(request))
+        val result = Await.result(connector.getUpscanFormData(request), timeout)
```

Comment:
if you extend  `with ScalaFutures` you won't need this `Await.result`

Reply:
done